code/modules/spacepods/spacepod.dm

```diff
@@ -54,12 +68,38 @@
 	var/move_delay = 2
 	var/next_move = 0
 
+/obj/spacepod/proc/apply_paint(mob/user as mob)
+	var/part_type
+	var/part = input(user, "Choose part", null) as null|anything in list("Lights","Rim","Paint","Windows")
+	if(part == "Lights")
```

Comment:
As of right now they'd have to be numbers arbitrarily, but I could add more defines if err, that's fine.

`	var/part_type = input(user, "Choose part", null) as null|anything in list(LIGHT,WINDOW,RIM,PAINT)
	if(!part_type)
		return
	var/coloradd = input(user, "Choose a color", "Color") as color
	colors[part_type] = coloradd`
That's what I was thinking of doing, which yeah would need numbers. Or I could just change it to a switch like DZD said